### PR TITLE
Not much

### PR DESCRIPTION
**Pull request by AlCalzone**
see comments

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,15 +2,11 @@ const {PluginBase} = require('@iobroker/plugin-base');
 
 class SentryPlugin extends PluginBase {
 
-    constructor(settings) {
-        super(settings);
-    }
-
```
Comment (AlCalzone) [inline index.js]: Unnecessary

File: index.js [hunk 2]
```diff
@@ -2,15 +2,11 @@ const {PluginBase} = require('@iobroker/plugin-base');
...
     /**
      * Register and initialize Sentry
      *
-     * @param pluginConfig {object} plugin configuration from config files
-     * @param callback {function} callback when done, signature "(err, initSuccessful)". On err or initSuccessful===false the plugin instance will be discarded
+     * @param {Record<string, any>} pluginConfig plugin configuration from config files
+     * @param {import('@iobroker/plugin-base').InitCallback} callback Will be called when done
```
Comment (AlCalzone) [inline index.js]: Requires the other PR on plugin-base to be merged

File: index.js [hunk 3]
```diff
@@ -23,7 +19,7 @@ class SentryPlugin extends PluginBase {
         }
         // Require needed tooling
         this.Sentry = require('@sentry/node');
-        this.SentryIntegrations = require('@sentry/integrations');
+        const SentryIntegrations = require('@sentry/integrations');
```
Comment (AlCalzone) [inline index.js]: No need to create a property for that IMO

File: index.js [hunk 4]
```diff
@@ -109,15 +105,6 @@ class SentryPlugin extends PluginBase {
...
 
-    /**
-     * Destroy plugin instance
-     *
-     * @return {boolean} true/false depending on if exit was successful/is supported
-     */
-    destroy() {
-        return true;
-    }
-
```
Comment (AlCalzone) [inline index.js]: Also unnecessary, since this is the default behavior.

File: index.js (hunk 1 above)
Comment (Apollon77) [reply index.js]: I was unsure :-)

File: index.js (hunk 3 above)
Comment (Apollon77) [reply index.js]: right

File: index.js (hunk 4 above)
Comment (Apollon77) [reply index.js]: right, was different before :-)

**Pull request merged**